lib/archethic/bootstrap/transaction_handler.ex

```diff
@@ -36,7 +36,10 @@ defmodule Archethic.Bootstrap.TransactionHandler do
   @spec do_send_transaction(list(Node.t()), Transaction.t()) ::
           :ok
   defp do_send_transaction([node | rest], tx) do
-    case P2P.send_message(node, %NewTransaction{transaction: tx}) do
+    case P2P.send_message(node, %NewTransaction{
+           transaction: tx,
+           welcome_node: Crypto.first_node_public_key()
```

Comment:
At this point, the real welcome node key will be the the one which will receive the transaction `node.first_public_key`. Because here the node act as a client.